src/components/TodoItem.jsx

```diff
@@ -0,0 +1,33 @@
+import { useState } from 'react'
+import { AiOutlineClose } from 'react-icons/ai';
+import { useDispatch } from 'react-redux';
+import { deleteTodo, toggleTodoStatus, updateTodoTitle } from '../redux/features/todos';
+
+function TodoItem({title, done, id}) {
+    const [editing, setEditing] = useState(false);
+    const [value, setValue] = useState(title);
+    const dispatch = useDispatch();
+
+    const handleUpdateTitle = () =>{
+        value && dispatch(updateTodoTitle({id, title: value}));
+        setEditing(false)
+    }
+
+  return (
+    <div className='flex items-center justify-between border-b border-gray-300 last:border-0 py-3'>
```

Comment:
You used single quotes for the className but for the input, you are using double-quotes. You should be consistent.